### ISSUE TITLE
[Bug] about:config isn't enabled in Fenix nightly

### ISSUE THREAD

**Issue by jamienicol** [mozilla-mobile/fenix#5529]
## Steps to reproduce

Navigate to about:config

### Expected behavior

the about:config page opens

### Actual behavior

Error message: cannot complete request

### Device information

* Android device: Pixel 2, android 10
* Fenix version: "Nightly 190924 12:53" (Build #12671259)
📦: 14.0.0, 186fa2396
🦎: 71.0a1-20190923094800

**Comment by pocmo** [comment]
`about:config` needs to be explicitly enabled in GeckoView now:
https://mozilla.github.io/geckoview/javadoc/mozilla-central/org/mozilla/geckoview/doc-files/CHANGELOG

```
Added GeckoRuntimeSettings.Builder#aboutConfigEnabled to control whether or not about:config should be available. (bug 1540065)
```

https://bugzilla.mozilla.org/show_bug.cgi?id=1540065

**Comment by hakkikaancaliskan** [comment]
@pocmo but it should be enabled for nightly builds, right?

**Comment by AndiAJ** [comment]
Hi, verified as fixed on Nightly Build #12700627 from 27/9 using the following devices:
• Google Pixel 3a (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0)
• Samsung Galaxy S7 (Android 7)
• OnePlus A3 (Android 6.0.1)
• LG Nexus 4 (Android 5.1.1)

► Pic
![rsz_screenshot_20190927-105710](https://user-images.githubusercontent.com/51314259/65753274-56e73080-e117-11e9-9ee2-3c72733dd88c.png)
